package com.mydigipay.challenge.data.network.error

class UnAuthorizedException : RuntimeException()